scripts/spack/configs/toss_4_x86_64_ib/spack.yaml

```diff
@@ -291,13 +291,19 @@ spack:
         prefix: /usr/tce/packages/python/python-3.9.12
 
     # Globally lock version of third party libraries
+    axom:
+      require: "@0.9.0.1"
     camp:
-      require: "@2023.06.0"
+      require: "@2024.02.0"
     conduit:
-      require: "@0.8.8~shared~test~examples~utilities"
+      require: "@0.9.1~shared~test~examples~utilities"
     hdf5:
       variants: ~shared~mpi
+    mfem:
+      require: "@4.6.2.4"
     raja:
-      require: "@2023.06.1~shared~examples~exercises"
+      require: "@2024.02.0~shared~examples~exercises"
+    tribol:
+      require: "@0.1.0.15~examples~tests"
     umpire:
-      require: "@2023.06.0~shared~examples"
+      require: "@2024.02.0~shared~examples~werror"
```

Comment:
Why not have this information in the Serac spack `package.py`?

Reply:
I only put hard requirements in the `package.py` because it is used by other projects. I did put the version requirement in the `package.py` due to it being a requirement, but `~werror` was fine before and only failed due to a weird set of compilers.